CONTRIBUTING.md

```diff
@@ -122,6 +122,10 @@ _[Read more about this](https://github.com/meilisearch/integration-guides/blob/m
 This project integrates a tool to create automated changelogs.<br>
 _[Read more about this](https://github.com/meilisearch/integration-guides/blob/main/guides/release-drafter.md)._
 
+### Labels
+
+To chose the proper labels reade the [labels guide](https://github.com/meilisearch/integration-guides/blob/main/guides/release-drafter.md#how-does-the-release-drafter-work)). Labels impact the generated version for the next release.
+
```

Comment:
There is already the explanation in the guide, no need to make the CONTRIBUTING longer. Plus, the `label` title is out of context for someone who doesn't know the release drafter -> they don't know why we are talking about labels.
I would rather keep all the information of the release drafter in https://github.com/meilisearch/integration-guides/blob/main/guides/release-drafter.md 🙂